## Introdução
### O que vamos aprender?
Você já aprendeu a tratar requisições assíncronas utilizando o .then(). Hoje você aprenderá uma outra forma de fazer isso, utilizando async/await. Também mostraremos como tratar possíveis erros nas nossas requisições assíncronas, utilizando o bloco try/catch.

Além disso, você aprenderá a utilizar o async/await para realizar testes que possuem requisições assíncronas.


Você será capaz de:
Escrever códigos assíncronos utilizando async/await;
Utilizar o try/catch para tratar erros;
Escrever testes para códigos assíncronos utilizando async/await.
Por que isso é importante?
Códigos assíncronos são comuns em JavaScript. Você certamente vai precisar escrevê-los com frequência, por exemplo, quando você realiza uma requisição para uma API. Muitas vezes essas APIs podem nos retornar erros, seja por termos enviado algum parâmetro errado, ou mesmo por falha no servidor que hospeda a API.

Para a nossa aplicação não quebrar quando isso ocorrer, precisamos tratar os erros enviados pela API. Além disso, precisamos aprender a testar nossas aplicações que utilizam requisições assíncronas.

